### PR TITLE
Adding links to what-is-new.md

### PR DESCRIPTION
**Pull request by rickarm**
Directing people to links within /learn and other places in the site

### CODE REVIEW

**Comment by TheAlphaNerd** [comment]
did you mean to close this?

**Comment by rickarm** [comment]
well i was trying to move this PR to dev.  but if you can accept on master that would be nice :)

**Comment by TheAlphaNerd** [comment]
technically it makes no difference which branch the pr is sent to. I always manually merge them anyways :D

**Comment by rickarm** [comment]
Coolio. Merge away!

File: app/content/what-is-new.md [hunk 1]
```diff
@@ -61,7 +62,7 @@ We've redesigned our sizing API to increase flexibility. Developers can now fine
 We've built Famous with extensibility in mind. In particular, we decoupled our scene graph from our rendering pipeline by using a draw command buffer. This paves the way for new integrations and allows for frameworks to be built on top of our drawing API. 
 
 
-###MIT License:
+###[MIT License](https://github.com/famous/engine):
```
Comment (TheAlphaNerd) [inline app/content/what-is-new.md]: do you think this should be a link to our license?

Comment (TheAlphaNerd) [inline app/content/what-is-new.md]: I'm going to just move ahead with the change and we can revert

**Comment by TheAlphaNerd** [comment]
landed as 0572809

File: app/content/what-is-new.md (hunk 1 above)
Comment (rickarm) [inline app/content/what-is-new.md]: Let's leave for now.  Can always change back.

**Comment by rickarm** [comment]
Not seeing the updates on the site content.  Any reason why not?

**Comment by TheAlphaNerd** [comment]
it is up on staging... not sure why master didn't update

**Comment by TheAlphaNerd** [comment]
will be taking this up with marianna today

**Comment by Nefry** [comment]
Deployed to productions. Now there are separate jobs for develop and master so we will not miss changes.

**Pull request closed without merging**